prod/README.md

```diff
@@ -38,7 +38,12 @@ docker run -v ~/git/miqa/prod:/etc/nginx/conf.d:ro --network=host nginx
 ```
 
 ## Pull the evaluation models from git LFS
-TODO
+```
+cd miqa/learning/models
+git-lfs fetch
```

Comment:
Do we have `git lfs install` mentioned anywhere?

Reply:
Added in [1547442](https://github.com/OpenImaging/miqa/pull/388/commits/1547442cf85cfa61499913972a9d27b10c9281a9)